### DISCUSSION TITLE
Alert Boxの中にCode Blockを書くとはみ出す

### DISCUSSION THREAD

**Discussion by sor4chi** [increments/qiita-discussions#269] [要望・アイデア・不具合]
仕様だったらごめんなさい🙏
Alert Boxの中にCode Blockを書くとはみ出してデザイン崩れ？のようになっている箇所を見つけましたので共有します。

発見場所: https://qiita.com/mpyw/items/37f053b322f24dde52e1
環境: M1 Mac Book Air / Google Chrome v103.0.5060.134

<img width="992" alt="スクリーンショット 2022-08-04 13 34 51" src="https://user-images.githubusercontent.com/80559385/182763989-badeab51-f76a-4f60-b658-53996b3394ec.png">

**Comment by tanaka-asami415** [comment]
@sor4chi 
ありがとうございます🙏
ノート記法でコードブロックをまだサポートしていないのですが、
 #252 こちらでも同様のご意見をいただいており、今後はサポートしたいと考えております🙇

**Reply by tanaka-asami415** [reply]
@sor4chi 
大変お待たせしました:bow:
こちらの修正が完了いたしました！

**Reply by sor4chi** [reply]
早急なありがとうございました🙇‍♂️
今後ともよろしくお願いします！
